app/javascript/api/authApiService.ts

```diff
@@ -35,7 +35,10 @@ export const createAccount = async (
     user,
     contact,
     locale: getCurrentLanguage(),
-    confirm_success_url: "http://localhost:3000/my-account", // TODO DAH-2566
+    confirm_success_url:
+      process.env.NODE_ENV !== "production"
+        ? "https://dahlia-full.herokuapp.com/my-account"
+        : "https://housing.sfgov.org/my-account",
```

Comment:
nitpick: why not make this an env variable rather than hardcoding based on the environment?

Reply:
I wasn't sure where we are at with adding more env variables - down to make it an env variable if that's what you advise

Reply:
In my experience I've found it more manageable to make these env to env variables an environment variable. But like you mentioned we haven't normed on how to approach these. For me this PR is out of scope of this discussion and looks good to me, we can discuss further in our eng [sync](https://docs.google.com/document/d/1k63F_iX3_EuyeikSghf8C143P9bMjJKMOdTGifxLU6Y/edit?tab=t.0)